package.json

```diff
@@ -23,6 +23,7 @@
 		"@wordpress/is-shallow-equal": "1.0.2",
 		"@wordpress/url": "1.1.0",
 		"@wordpress/wordcount": "1.0.2",
+		"ajv": "6.5.2",
```

Comment:
I didn't catch where this is used. Is it needed?

Reply:
It's strange to see this here. Could this have gotten in during a rebase?